aiohttp/web_fileresponse.py

```diff
@@ -37,6 +37,14 @@
 
 NOSENDFILE: Final[bool] = bool(os.environ.get("AIOHTTP_NOSENDFILE"))
 
+# File extensions for IANA encodings that will be checked in the order defined.
+# TODO(py311): Use StrEnum conversion of ContentCoding for key type and remove .value
+ENCODING_EXTENSION: Final[dict[str, str]] = {
```

Comment:
```suggestion
ENCODING_EXTENSION: Final[Dict[str, str]] = {
```

Reply:
Yeah I just educated myself on the historical differences between typing.* and built in `dict`, `list`, etc.  Thanks.